src/helper_generic.py

```diff
@@ -1,26 +1,30 @@
-import os
+"""This module is to perform generic oprations for threading."""
+"""and some conversion operations."""
+
 import socket
 import sys
 from binascii import hexlify, unhexlify
 from multiprocessing import current_process
-from threading import current_thread, enumerate
+import threading
 import traceback
 
 import shared
 from debug import logger
 import queues
 import shutdown
 
+
 def powQueueSize():
     curWorkerQueue = queues.workerQueue.qsize()
-    for thread in enumerate():
+    for thread in threading.enumerate():
         try:
             if thread.name == "singleWorker":
                 curWorkerQueue += thread.busy
-        except:
-            pass
+        except Exception:
+            print ""
```

Comment:
Maybe this can put an entry into the log or something, at INFO severity. In the past, there were threads being quickly created and destroyed, which could have caused an issue here without locking, but the current implementation launches all the threads at start and then keeps them running forever. There are minor exeptions like the UPnP thread, which can be turned on or off during runtime, but I don't see it as a problem here.